fe/fe-core/src/main/java/org/apache/doris/qe/ConnectContext.java

```diff
@@ -526,7 +526,7 @@ public QueryDetail getQueryDetail() {
     }
 
     public AuditEventBuilder getAuditEventBuilder() {
-        return auditEventBuilder;
+        return new AuditEventBuilder();
```

Comment:
do not new AuditEventBuilder() here, new AuditEventBuilder for prepared statement EXECUTE

Reply:
Thank you for your correction, I have modified this part by performing a deep copy of the `auditevent` object to prevent the statement recorded in the audit log from being overwritten.